src/dbnode/generated-source-files.mk

```diff
@@ -157,20 +157,6 @@ genny-map-storage-index-results: install-m3x-repo
 	# Rename generated map file
 	mv -f $(m3db_package_path)/src/dbnode/storage/index/map_gen.go $(m3db_package_path)/src/dbnode/storage/index/results_map_gen.go
 
-# Map generation rule for storage/index/AggregationValuesMap
-.PHONY: genny-map-storage-index-aggregation-values
```

Comment:
why delete this?

Reply:
I think this one was github weirdness, right? Or still a valid comment?